### ISSUE TITLE
`v fmt` doesn't work properly with src directory

### ISSUE THREAD

**Issue by ChewingBever** [vlang/v#16288]
**V version:** V 0.3.2 135b1cc
**OS:** linux, "EndeavourOS Linux"

**What did you do?**
I tried migrating my repository to the newly supported `src` directory structure.

**What did you expect to see?**
Everything to work just fine, including `v fmt`.
 
**What did you see instead?**
`v fmt` tries to prepend `src.` to every module and import. It seems to be unaware that `src` should be the root of the code, and not the directory where `v.mod` is located.

**Minimal reproducible example**
`v.mod`: just an empty placeholder file
`src/main.v`:
```v
import somemod

fn main() {
	somemod.something()
}
```
`src/somemod/somemod.v`:
```v
module somemod

pub fn something() {
	println('hi')
}
```
Result of running `v fmt -diff .`:
```
jjr@tuur /t/test> v fmt -diff .
--- /tmp/test/src/main.v        2022-11-02 08:32:56.459769008 +0100
+++ /tmp/v_1000/vfmt_01GGVME2HFXZK5X3D53NJ8QR50_main.v  2022-11-02 08:38:33.382605270 +0100
@@ -1,3 +1,3 @@
-import somemod
+import src.somemod
 
 fn main() {
```

**Comment by merlindiavova** [comment]
Is this still an error? Seems to be working at least for the codes I have

**Comment by ArtemkaKun** [comment]
Fixed in #17633
